Add abstract (#167)
myyoda/principles-paper: main.tex

\anchoredsection{abstract}{Abstract}

Scientific claims now rest on stacks of code, data, and environments; yet the provenance of how they are used together is routinely incomplete, scattered, or lost, which undermines rigor, reproducibility, reusability, and efficiency.
Researchers across domains have converged toward analogous patterns, and tooling has matured to meet them.

> What is still missing is a shared vocabulary for how a research object, a complete unit of data, code, and metadata, should be structured and managed so it can be re-executed, extended, and redistributed; FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.

Author: ```suggestion
What is still missing is a shared vocabulary for how a research object, a complete unit of data, code, and metadata, should be structured and managed so it can be re-executed, extended, and redistributed.
FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.
```

\anchoredsection{abstract}{Abstract}

> Scientific claims now rest on stacks of code, data, and environments; yet the provenance of how they are used together is routinely incomplete, scattered, or lost, which undermines rigor, reproducibility, reusability, and efficiency.

Reviewer: ```suggestion
Researchers across the computational sciences increasingly depend on the interplay between code, data, and software environments.
Yet the records of how they are used together is routinely incomplete, scattered, or lost.
This undermines rigor, reproducibility, reusability, and efficiency.
```

Plenty of scientific claims, both new and old, that don't rely at all on software (many wet lab folks still write things down with pencils and paper)

Also some people might see 'provenance' in the first two sentences and run away

Author: ```suggestion
Scientific claims now frequently rest on stacks of code, data, and environments.
Yet the records of how they are used together is routinely incomplete, scattered, or lost.
This undermines rigor, reproducibility, reusability, and efficiency.
```

Researchers across domains have converged toward analogous patterns, and tooling has matured to meet them.

> What is still missing is a shared vocabulary for how a research object, a complete unit of data, code, and metadata, should be structured and managed so it can be re-executed, extended, and redistributed.

Reviewer: ```suggestion
What is still missing is a shared vocabulary for how a research object as a complete unit of data, code, and metadata, should be structured and managed so it can be re-executed, extended, and redistributed.
```

FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.

> Building on the YODA and VAMP frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).

Reviewer: ```suggestion
Building on previous frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).
```

I don't think we should call out two unpublished (manuscript-wise) and rather domain-specific frameworks in the abstract. I think it would only distract or confuse

> Together they guide how researchers produce and evaluate research objects so the resulting work is rigorous, reproducible, and reusable.

Reviewer: ```suggestion
Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build on.
```

less repetitive and more tangible outcomes

> We frame each principle as a spectrum, so adoption is incremental and starts from existing practice.

Reviewer: ```suggestion
We frame each principle as a spectrum so that adoption is incremental and starts from existing practice.
```

> Developing conventions and increasingly capable tooling make rigor, reproducibility, reusability, and efficiency more achievable than ever before, and this shared vocabulary gives researchers, collaborators, and reviewers a common language, common goals, and a common direction.

Reviewer: ```suggestion
This shared vocabulary gives researchers, collaborators, and reviewers a common language, aligned goals, and a path forward together.
```

\anchoredsection{abstract}{Abstract}

> Scientific claims now frequently rest on stacks of code, data, and environments.

Reviewer: "stacks of code" just reads so oddly; please find a different wording to serve as the first line a reader sees

Author: Aw, I liked the flavor of that, but if it feels like too much jargon I'd be fine with:

```suggestion
Scientific claims now frequently rest on the interplay of code, data, and environments.
```

Reviewer: not even jargon since you're not using the formal computer science meaning; code could technically form a stack of paper if you printed it on paper, but why would anyone do that XD

> Scientific claims now frequently rest on the interplay of code, data, and environments.

Reviewer: ```suggestion
Scientific claims increasingly rely upon the interplay of code, data, and environments.
```

flows more naturally?

Yet the records of how they are used together is routinely incomplete, scattered, or lost.
This undermines rigor, reproducibility, reusability, and efficiency.
Researchers across domains have converged toward analogous patterns, and tooling has matured to meet them.
What is still missing is a shared vocabulary for how a research object as a complete unit of data, code, and metadata, should be structured and managed so it can be re-executed, extended, and redistributed.
FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.
Building on previous frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).
Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build on.
We frame each principle as a spectrum so that adoption is incremental and starts from existing practice.

> Developing conventions and increasingly capable tooling make rigor, reproducibility, reusability, and efficiency more achievable than ever before, and this shared vocabulary gives researchers, collaborators, and reviewers a common language, common goals, and a common direction.

Reviewer: 'common' repeated 3 times

Author: I'm gonna shop this sentence (maybe break it up). "increasingly capabale tooling" was intended to be a nod to AI, but too subtle (the rest of the tooling is also increasingly capable), and I think I agree that its ok to say it explicitly. 

triple common was deliberate repetition for effect, but I did suspect you'd hate it ;)

Reviewer: 'duplication is evil' I think I'm supposed to say

Author: ```suggestion
Developing conventions, increasingly capable tooling, and AI agents make rigor, reproducibility, reusability, and efficiency more achievable than ever before.
This shared vocabulary augments all of them, giving researchers, collaborators, reviewers, and the agents working with them a common language, concrete goals, and an aligned direction.
```

Reviewer: The way as phrased still takes a couple re-reads before the proose fits

I instinctively interpret the first two words as meaning the _act_ of developing conventions

\anchoredsection{abstract}{Abstract}

> Scientific claims increasingly rely upon the interplay of code, data, and environments.

Reviewer 2: ```suggestion
Scientific claims increasingly rely upon the interplay of code, data, and computational environments.
```

Because of this the first sentence of the paper. I think we can be more specific since environments can mean many kind of environments.

> Yet the records of how they are used together is routinely incomplete, scattered, or lost.

Reviewer 2: ```suggestion
Yet the records of how they are used together is often incomplete, scattered, or lost.
```

"Routinely" has some implication of there is a routine, by design or not, to make this happen. I feel "often" more apppriate.

This undermines rigor, reproducibility, reusability, and efficiency.

> Researchers across domains have converged toward analogous patterns, and tooling has matured to meet them.

Author: ```suggestion
```
Cutting here, will come back farther down stronger and applied to STAMPED after thats introduced

> Previous approaches to this govern digital objects in general but leave the operational layer between them without a shared vocabulary for how a research object, as a complete unit, should be structured and managed so it can be re-executed, extended, and redistributed.

Reviewer 2: We are packing to much into one sentence.

The "this" in "previous approaches to this" is actually not clear.

> Previous approaches to this govern digital objects in general but leave the operational layer between them without a shared vocabulary for how a research object, as a complete unit, should be structured and managed so it can be re-executed, extended, and redistributed.

Reviewer: ```suggestion
Previous approaches to this issue govern digital objects in general but leave the operational layer between them without a shared vocabulary for how a research object, as a complete unit, should be structured and managed so it can be re-executed, extended, and redistributed.
```

```suggestion
Previous approaches to this problem govern digital objects in general but leave the operational layer between them without a shared vocabulary for how a research object, as a complete unit, should be structured and managed so it can be re-executed, extended, and redistributed.
```

Pick one

> Previous approaches to this govern digital objects in general but leave the operational layer between them without a shared vocabulary for how a research object, as a complete unit, should be structured and managed so it can be re-executed, extended, and redistributed.

Author: ```suggestion
Previous approaches have improved how digital objects are governed in general but do not specify how research objects ought to be structured and managed so they can be re-executed and extended.
The community is missing is a shared vocabulary for this operational layer.
```

The community is missing a shared vocabulary for this operational layer.

> Building on recurrent practices and convergent patterns across disciplines, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).

Reviewer 3: going from "principles:" into "-ed" properties form sounded odd to me.

Reviewer 3: and our sub-sections follow the "principles" form, so I would have kept it aligned 

```suggestion
Building on recurrent practices and convergent patterns across disciplines, we formalize seven principles: Self-containment, Tracking, Actionability, Modularity, Portability, Ephemerality, and Distributability (STAMPED).
```

> Building on recurrent practices and convergent patterns across disciplines, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).

Author: ```suggestion
Building on recurrent practices and convergent patterns across disciplines, we formalize seven principles: Self-containment, Tracking, Actionability, Modularity, Portability, Ephemerality, and Distributability (STAMPED).
```

Building on previous frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).
Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build on.

> We frame each principle as a spectrum so that adoption is incremental and starts from existing practice.

Author: ```suggestion
We frame each principle as a spectrum so that adoption is incremental and starts from existing practice.
The independent convergence across disciplines toward these same principles over decades is evidence they name recurrent patterns, not just another framework.
We back each principle with normative requirements, an interactive checklist, and a map of enabling tools.
```

> Developing conventions, increasingly capable tooling, and AI agents make rigor, reproducibility, reusability, and efficiency more achievable than ever before.

Reviewer: The way as phrased still takes a couple re-reads before the proose fits

I instinctively interpret the first two words as meaning the act of developing conventions

```suggestion
Emerging conventions, increasingly capable tooling, and AI agents are making rigor, reproducibility, reusability, and efficiency more achievable than ever before.
```

```suggestion
With maturing conventions, increasingly capable tooling, and the arrival of AI agents, rigor, reproducibility, reusability, and efficiency are within closer reach than ever.
```

Reviewer: Pick which one you like here

All 3 forms also contain a myriad of commas, but it's 3 combined lists so IDK how to avoid

Author: I like this!

> Developing conventions, increasingly capable tooling, and AI agents make rigor, reproducibility, reusability, and efficiency more achievable than ever before.

Author: ### 
```suggestion
Emerging conventions, increasingly capable tooling, and AI agents make rigor, reproducibility, reusability, and efficiency more achievable than ever before.
```

> As conventions mature, tooling improves, and agents become increasingly capable participants in research workflows, the goals of rigor, reproducibility, reusability, and efficiency are becoming more attainable.

Author: ```suggestion
As conventions mature, tooling improves, and AI agents become increasingly involved in research workflows, the goals of rigor, reproducibility, reusability, and efficiency are becoming more attainable.
```

> This shared vocabulary augments all of them, giving researchers, collaborators, reviewers, and the agents working with them a common language, concrete goals, and an aligned direction.

Author: ```suggestion
As conventions mature, tooling improves, and agents become increasingly capable participants in research workflows, the goals of rigor, reproducibility, reusability, and efficiency are becoming more attainable.
STAMPED gives researchers, collaborators, reviewers, and agents a common language, concrete goals, and an aligned direction for making computational research more durable.
```

> STAMPED gives researchers, collaborators, reviewers, and agents a common language, concrete goals, and an aligned direction for making computational research more durable.

Reviewer 3: someone will be the first to coin word

```suggestion
STAMPED gives researchers, collaborators, reviewers, and aigents a common language, concrete goals, and an aligned direction for making computational research more durable.
```

will it be us? ;)

Author: lol, i think it might be you but not here ;)

\anchoredsection{abstract}{Abstract}

Scientific claims increasingly rely upon the interplay of code, data, and computational environments.

> Yet the records of how they are used together is often incomplete, scattered, or lost.

Author: ```suggestion
Yet the records of how they are used together are often incomplete, scattered, or lost.
```

> Yet the records of how they are used together are often incomplete, scattered, or lost.

Reviewer: ```suggestion
Yet the record of how they are used together is often incomplete, scattered, or lost.
```

> Yet the records of how they are used together are often incomplete, scattered, or lost.

Author: ```suggestion
Yet the record of how they are used together is often incomplete, scattered, or lost.
```

This undermines rigor, reproducibility, reusability, and efficiency.

> Previous approaches have improved how digital objects are governed in general but do not specify how research objects ought to be structured and managed so they can be re-executed and extended.

Author: ```suggestion
Previous approaches have improved the governance of digital objects but do not specify how research objects ought to be structured and managed so they can be re-executed and extended.
```

> What is still missing is a shared vocabulary for how a research object as a complete unit of data, code, and metadata, should be structured and managed so it can be re-executed, extended, and redistributed.

Reviewer: ```suggestion
What is missing is a shared vocabulary for how a research object as a complete unit ought to be structured and managed so it can be re-executed, extended, and redistributed.
```

'still' might sound too accusatory to the community

Also shortening duplication of the code/data/extra list which is repeating many times now

Author: ```suggestion
What is missing is a shared vocabulary for the operational layer; how a research object as a complete unit ought to be structured and managed so it can be re-executed, extended, and redistributed.
```

> The community is missing is a shared vocabulary for this operational layer.

Author: ```suggestion
The community is missing a shared vocabulary for this operational layer.
```

> FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.

Reviewer 2: I feel that the transition from the last sentence to this one is a bit abrupt even though I can see the relation (with some effort).

> FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.

Reviewer: ```suggestion
The previous FAIR and FAIR4RS approaches apply to digital objects in general, but not to the operational layer which connects their creation to ???.
```

> FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.

Reviewer: ```suggestion
The previous FAIR and FAIR4RS approaches apply to digital objects in general, but not to the operational layer between them.
What is still missing is a shared vocabulary for how a research object as a complete unit ought to be structured and managed so it can be re-executed, extended, and redistributed.
```

> FAIR and FAIR4RS apply to digital objects in general, but not to this operational layer.

Reviewer: ```suggestion
Previous approaches to this govern digital objects in general but leave the operational layer between them without a shared vocabulary for how a research object, as a complete unit, should be structured and managed so it can be re-executed, extended, and redistributed.
```

> Building on past frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).

Author: ```suggestion
Building on recurrent practices and convergent patterns across disciplines, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).
```

> Building on previous frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).

Reviewer: ```suggestion
Building on past frameworks, we formalize seven principles: Self-contained, Tracked, Actionable, Modular, Portable, Ephemeral, and Distributable (STAMPED).
```

> Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build on.

Author: ```suggestion
Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build upon.
```

> Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build on.

Reviewer: ```suggestion
Together they give researchers guidelines for building and assessing research objects that others can trust, rerun, and build upon.
```

We frame each principle as a spectrum so that adoption is incremental and starts from existing practice.

> We show that independent convergence across disciplines over decades is evidence that these principles name recurrent patterns.

Author: ```suggestion
```

> The independent convergence across disciplines toward these same principles over decades is evidence they name recurrent patterns, not just another framework.

Author: ```suggestion
We show that independent convergence across disciplines over decades is evidence that these principles name recurrent patterns.
```

> We back each principle with normative requirements, an interactive checklist, and a map of enabling tools.

Author: ```suggestion
We support each principle with normative requirements, an interactive checklist, and a map of enabling tools.
```

> With maturing conventions, increasingly capable tooling, and the arrival of AI agents, rigor, reproducibility, reusability, and efficiency are within closer reach than ever.

Author: ```suggestion
With maturing conventions, increasingly capable tooling, and the arrival of AI agents, the scientific goals of rigor, reproducibility, reusability, and efficiency are within closer reach than ever.
```
